PSICover/PSICover.cs

```diff
@@ -146,8 +147,80 @@ class Analyzer {
       ExecProgram ($"{Dir}/{RunExe}", "");
    }
 
+   double GetFileCoverage (string file, int blockCount, int hitBlocks) {
+      var blocks = mBlocks.Where (a => a.File == file);
+      ulong[] hits = File.ReadAllLines ($"{Dir}/hits.txt").Select (ulong.Parse).ToArray ();
+      var p = hitBlocks * 100.0 / blockCount;
+      return Math.Round (p, 1);
+   }
+
+   void GenTable () {
+      ulong[] hits = File.ReadAllLines ($"{Dir}/hits.txt").Select (ulong.Parse).ToArray ();
+      int cBlocks = mBlocks.Count, cHit = hits.Count (a => a > 0);
+      double percent = Math.Round (100.0 * cHit / cBlocks, 1);
+
+      string text = $$"""
+            <h2>PSI Tests
+            Total coverage: {{percent}}%</h2>
+            <table>
+               <tr>
+               <th>Module</th>
+               <th>File</th>
+               <th>Total blocks</th>
+               <th>Hit blocks</th>
+               <th>Coverage</th>
+               </tr>
+            """;
+      var modules = mBlocks.Select (a => a.Module).Distinct ().ToArray ();
+      foreach (var module in modules) {
+         text += $$"""

```

Comment:
Use StringBuilder or string.Join as string is readonly and we are creating more temporary string.